search/mongo_search.go

```diff
@@ -455,7 +455,7 @@ func (m *MongoSearcher) createQuantityQueryObject(q *QuantityParam) bson.M {
 				bson.M{"unit": ci(q.Code)},
```

Comment:
As discussed with @cjduffett, he decided to leave this in since units (like `lbs`) probably _should_ be case-insensitive (as opposed to actual codes in code systems).